### PR TITLE
Manually add EOD to LTS notes

### PR DESCRIPTION
**Pull request by TheMeaningfulEngineer**

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]
All right :+1: Thanks

You need to amend your commit following the new formatting, see the CI error message.

File: 302.Release-information/01.Release-schedule/docs.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ The lifecycle of an LTS release begins on the release date and ends one year lat
 <!--AUTOVERSION: "LTS releases: %"/lts -->
 At this time, we support the following LTS releases: 3.3, 3.0.
```
Comment (kacf) [inline 302.Release-information/01.Release-schedule/docs.md:16]: This is now wrong.

Comment (lluiscampos) [reply 302.Release-information/01.Release-schedule/docs.md:16]: So it is the table below, then. I thought the idea was to manually amend after 3.4 release. But I am okey with removing it now, it is already EOL indeed.

**Comment by TheMeaningfulEngineer** [comment]
@lluiscampos 
Updated the commit signature.
Seems to be failing for unrelated reasons now:

```
ERROR: for mender-device-auth  Head "https://registry.mender.io/v2/mendersoftware/deviceauth-enterprise/manifests/staging": no basic auth credentials
ERROR: for mender-deployments  Head "https://registry.mender.io/v2/mendersoftware/deployments-enterprise/manifests/staging": no basic auth credentials
ERROR: for mender-workflows-server  Head "https://registry.mender.io/v2/mendersoftware/workflows-enterprise/manifests/staging": no basic auth credentials
ERROR: for mender-useradm  Head "https://registry.mender.io/v2/mendersoftware/useradm-enterprise/manifests/staging": no basic auth credentials
ERROR: for mender-inventory  Head "https://registry.mender.io/v2/mendersoftware/inventory-enterprise/manifests/staging": no basic auth credentials
```

**Pull request merged**